package.json

```diff
@@ -89,13 +89,13 @@
     "clean": "rm -rf dist",
     "clean:compiled-migrations": "rm -rf dist_migrations",
     "db:create": "node_modules/.bin/sequelize db:create && yarn build-migrations",
-    "db:migrate": "sequelize db:migrate",
+    "db:migrate": "yarn build-migrations && sequelize db:migrate",
```

Comment:
PRAISE OUR SAVIOR AND LORD JESUS CHRIST

Reply:
jajajaj. Igual falta meter el cambio en el repo de deploy

Reply:
Recien pushee a master el cambio en el repo de deploy

Reply:
y borre el script: `db:staging:migrate` . Asique ahora se migra con `db:migrate`